lint/yodaStyleExpr_checker.go

```diff
@@ -42,6 +47,16 @@ func (c *yodaStyleExprChecker) isConstExpr(expr ast.Expr) bool {
 
 func (c *yodaStyleExprChecker) warn(expr *ast.BinaryExpr) {
 	e := astcopy.BinaryExpr(expr)
+	switch expr.Op {
+	case token.LSS:
+		e.Op = token.GTR
```

Comment:
@fexolm  is our expert in these things. :3 
`>` is not valid inverse of `<`. You need `>=` here.

Reply:
Same for other operations. You may want to build "truth table" if unsure.